src/style.css

```diff
@@ -1,3 +1,70 @@
+html {
+  font-family: Roboto, sans-serif;
+
+  --active-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  height: 60px;
+  position: relative;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+}
+
+a:hover {
+  color: var(--active-color);
+}
+
+.is-active {
+  color: var(--active-color);
+}
+
+a.is-active::after {
+  display: block;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  content: '';
+  background: #00acdc;
```

Comment:
Consider using the CSS variable '--active-color' for the background property in 'a.is-active::after' to maintain consistency and make it easier to update the color scheme. Replace 'background: #00acdc;' with 'background: var(--active-color);'.